modules/ROOT/pages/design-migrating-ms.adoc

```diff
@@ -0,0 +1,108 @@
++
+*Result:* The mocking service is switched off and then on. The format of the base URI that the mocking service generates now differs from the format of the previous base URI.
++
+Version 1 of the mocking service used base URIs that began with `https://mocksvc.mulesoft.com`.
++
+Version 2 uses base URIs that begin with `https://anypoint.mulesoft.com/mocking/api/v1/links/`.
+
+. If any third-party tools are testing endpoints in the API specification for that project, replace the previous base URI with the new base URI in the links that those tools are using.
+
+== New Features in Version 2 That You Can Use in API Designer
+
+Version 2 of the mocking service, which was released on January 10, 2019, introduced the following list of features that you can use in API Designer. For a full list of features, see the https://anypoint.mulesoft.com/exchange/org.mule.examples/mocking-service-2-api/version/v1/pages/Getting%20Started/[Getting Started page for the Mocking Service 2 API]. (The link requires that you authenticate to Anypoint Platform.)
+
+=== Support for OAS 2.0
+
+You can now test endpoints that are in API specifications that are written in OAS 2.0.
+
+
+=== Behavioral headers
+
+Add headers to your simulated calls to methods, changing the behavior of the mocking service in response to those calls The following new headers are available. See xref::apid-behavioral-headers.adoc[Add Behavioral Headers to Simulated API Calls] for full details.
+
+* MS2-Delay: Indicates how many milliseconds the request should take to answer.
+* MS2-Error-Level: Indicates which range of error status codes the mocking service is restricted to use.
+* MS2-Error-Rate: Indicates the probability of getting an error status code as a response.
+* MS2-Randomize: When this header is set to true, the mocking service auto-generates a payload response if no example is defined for the response.
+* MS2-Status-Code: Specifies the status code or codes that the mocking service can choose from for the return message.
+* MS2-Strict-Model-Validations: When you set this header to false, you can use the mocking service even when the API specification contains errors
+
+=== Performance improvements
+
+The mocking service uses fewer resources, but responds to requests in less time on average.
+
+
+=== Authenticated Access to API Specifications That are Shared on Exchange
+
+Only Anypoint Platform users can simulate calls to your API specification after it is published to a private instance of Exchange, whether they are simulating calls in Exchange or using third-party tools. The mocking service now uses a base URI in Exchange that differs from the one that you can test with in API Designer.
+
+If a person is in Exchange and trying to simulate a call to a method, that person is prompted in API Designer to supply OAuth 2.0 credentials before sending a request.
```

Comment:
I think the API Designer prompt part is wrong. If a person is in Exchange and use the console to simulate the call, it will just work (as Exchange will send the Authorization header under the hood). The thing is with a third party tool as it is explained in the following paragraph

Reply:
Yes, I mangled that, too. Thanks.